docs/products/access/api-reference/config-api/service-providers-api.md

```diff
@@ -0,0 +1,333 @@
+# Service Providers API
+
+This allows the creation of new Service Provider via a REST API.
+It can be utilized in scripts to add, list, edit, or delete service providers.
+
+All endpoints are protected with API client. 
+It requires an API client with the `onegini_api_config` scope (Config API).
```

Comment:
I'm wondering if it's clear to everyone what it means to create a Service Provider. It's more about registering an existing Service Provider than creating one. What is being created is obviously a resource representing such a service provider, but not the service provider itself.

Reply:
Good point. I changed it to create/update "Service Provider configuration"